types/musickit-js/MusicKit.Events.d.ts

```diff
@@ -6,7 +6,7 @@ declare namespace MusicKit {
         /**
          * A notification name indicating a change in the authorization status.
          */
-        authorizationStatusDidChange: unknown;
+        authorizationStatusDidChange: { authorizationStatus: AuthStatus["AUTHORIZED"] };
```

Comment:
The `authorizationStatusDidChange` is indeed fired with an `authorizationStatus` field:

![image](https://user-images.githubusercontent.com/81154791/147941724-c67d44b9-8240-4681-8fac-ec2cd3f48a3a.png)

and further analysis shows that the value of that field is a number